File: MyApp copy 2/app/src/main/java/com/example/myapp/DetailActivity.java
package com.example.myapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class DetailActivity extends AppCompatActivity {

    TextView textviewTitle,textviewDescription;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        bindView();
        setupIntentData();
    }

    private void setupIntentData() {
        Intent intent = getIntent();
        String title=intent.getStringExtra(AppConstant.TITLE);
        String description=intent.getStringExtra(AppConstant.DESCRIPTION);
        textviewTitle.setText(title);
        textviewDescription.setText(description);
    }

    private void bindView() {
        textviewTitle=findViewById(R.id.textViewTitle);
        textviewDescription=findViewById(R.id.textViewDescription);
    }
}

File: MyApp copy/app/src/main/java/com/example/myapp/DetailActivity.kt
package com.example.myapp

import android.accounts.AuthenticatorDescription
import android.nfc.Tag
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.tabs.TabLayout

class DetailActivity : AppCompatActivity(){

    lateinit var textviewTitle:TextView
    lateinit var textviewDescription:TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
        bindView()
        setupIntentData()
    }

    private fun setupIntentData() {
        //getIntent()
        val intent= intent
        val title=intent.getStringExtra(AppConstant.TITLE)
        val description=intent.getStringExtra(AppConstant.DESCRIPTION)
        //setText()
        textviewTitle.text = title
        textviewDescription.text = description
    }
    private fun bindView() {
        textviewTitle=findViewById(R.id.textViewTitle)
        textviewDescription=findViewById(R.id.textViewDescription)
    }


}